src/utlis/types.ts

```diff
@@ -2,6 +2,7 @@ export enum Variant {
     PRIMARY = 'primary',
     SECONDARY = 'secondary',
     TERTIARY = 'tertiary',
+    SPECIAL = 'special',
```

Comment:
Maybe call this `QUARTENARY = 'quartenary'` ? Since it's the fourth styling type